.cspell.json

```
@@ -384,6 +384,7 @@
     "burlywood",
     "butterworth",
     "buttord",
+    "ByteTrack",
```

@miursh I think `ByteTrack` is split into `Byte` and `Track` (normal words) when the spelling is checked.
So you meant `bytetrack`?

```suggestion
    "bytetrack",
```

---

Thanks. Either is ok for me.

---

@miursh Where did you encounter this error? Could you please give me the link?

---

Thank you. Okay then `bytetrack` is good!